tests/test_req2flatpak.py

```diff
@@ -95,6 +97,15 @@ def test_cli_with_reqs_as_args(self):
         build_module = json.loads(result.stdout)
         self.validate_build_module(build_module)
 
+    def test_cli_with_reqs_as_args_yaml(self):
+        """Runs req2flatpak in yaml mode by passing requirements as cmdline arg."""
+        args = ["--requirements"] + self.requirements
+        args += ["--target-platforms"] + self.target_platforms
+        args += ["--yaml"]
+        result = self._run_r2f(args)
+        build_module = yaml.load(result.stdout, yaml.Loader)
```

Comment:
I don't think specifying `loader` is needed. The rest looks good though.

Reply:
Without that, I get:
```
self = <tests.test_req2flatpak.EggTest testMethod=test_cli_with_reqs_as_args_yaml>

    def test_cli_with_reqs_as_args_yaml(self):
        """Runs req2flatpak in yaml mode by passing requirements as cmdline arg."""
        args = ["--requirements"] + self.requirements
        args += ["--target-platforms"] + self.target_platforms
        args += ["--yaml"]
        result = self._run_r2f(args)
>       build_module = yaml.load(result.stdout)
E       TypeError: load() missing 1 required positional argument: 'Loader'
```

Note: I'm not 100% sure I'm passing the correct thing, it does seem like this would be the sensible default...

Reply:
You are right, I had a look into the codebase. However [the docs](https://pyyaml.org/wiki/PyYAMLDocumentation#cb67) say something else.

Reply:
I found these yaml/pyyaml#576 and yaml/pyyaml.org#15. A comment in the latter suggests using `safe_load` since that doesn't need a loader.